applications/GeoMechanicsApplication/tests/C-Phi_reduction_process/MaterialParameters_stage1.json

```diff
@@ -0,0 +1,42 @@
+{
+   "properties": [{
+      "model_part_name":         "PorousDomain.Material_1",
+      "properties_id":           1,
+      "Material": {
+          "constitutive_law": {
+              "name"             :  "GeoLinearElasticPlaneStrain2DLaw" 
+          },
+          "Variables": {
+              "IGNORE_UNDRAINED"         :  true,
+              "K0_MAIN_DIRECTION"        :  1,
+              "YOUNG_MODULUS"            :  14.0E6,
+              "POISSON_RATIO"            :  0.3,
+              "DENSITY_SOLID"            :  2038.73598,
+              "DENSITY_WATER"            :  0.0,
+              "POROSITY"                 :  0.0,
+              "BULK_MODULUS_SOLID"       :  1.0e9,
+              "BULK_MODULUS_FLUID"       :  2.0e6,
+              "PERMEABILITY_XX"          :  4.5e-13,
+              "PERMEABILITY_YY"          :  4.5e-13,
+              "PERMEABILITY_XY"          :  0.0,
+              "DYNAMIC_VISCOSITY"        :  8.90e-7,
+              "UDSM_NAME"                :  "MohrCoulombUMAT.dll",
+              "IS_FORTRAN_UDSM"          :  true,
+              "NUMBER_OF_UMAT_PARAMETERS":   6,
+     	      "INDEX_OF_UMAT_PHI_PARAMETER": 4,
+     	      "INDEX_OF_UMAT_C_PARAMETER":   3,
+              "UMAT_PARAMETERS"          :  [ 5.384615e6,
+                                              0.3,
+                                              10000.0,
+                                              35.0,
+                                              0.0,
+                                              0.0 ],
+              "STATE_VARIABLES"          :  [ 0.0 ],
```

Comment:
Wasn't it possible to remove the other UDSM-related lines?

Reply:
I see. Thank you for explaining. I guess that we will need to address that at some point in the future, since it makes little sense to have these parameters here.